### PR TITLE
Add testdata for spaceranger

### PR DESCRIPTION
**Pull request by grst**
Adds two example datasets (reads subsampled from demo data from the 10x website). 

* Human ovarian cancer 1, FFPE direct placement, v1 chemistry
* Human brain cancer, FFPE cytassist, v2 chemistry

more details see README.

### CODE REVIEW

File: data/genomics/homo_sapiens/10xgenomics/spaceranger/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+# Input Files
+curl -O https://cf.10xgenomics.com/samples/spatial-exp/1.3.0/Visium_FFPE_Human_Ovarian_Cancer/Visium_FFPE_Human_Ovarian_Cancer_fastqs.tar
+curl -O https://cf.10xgenomics.com/samples/spatial-exp/1.3.0/Visium_FFPE_Human_Ovarian_Cancer/Visium_FFPE_Human_Ovarian_Cancer_image.jpg
+curl -O https://cf.10xgenomics.com/samples/spatial-exp/1.3.0/Visium_FFPE_Human_Ovarian_Cancer/Visium_FFPE_Human_Ovarian_Cancer_probe_set.csv
+
+# Extract
+tar xvf Visium_FFPE_Human_Ovarian_Cancer_fastqs.tar
+
+# Create subsampled dataset
```
Comment (klkeys) [inline data/genomics/homo_sapiens/10xgenomics/spaceranger/README.md:25]: is this done with ImageMagick? if so, then probably wise to make a note

Comment (klkeys) [reply data/genomics/homo_sapiens/10xgenomics/spaceranger/README.md:25]:
```suggestion
# Create subsampled dataset with ImageMagick
# https://imagemagick.org/index.php
```

**Review by klkeys — COMMENTED** [review]
this looks pretty good

only comment was about the `convert` command line tool, is it ImageMagick? many people won't know what it is, so I suggested adding a hyperlink to it

**Comment by grst** [comment]
Thanks for the review! Committed your suggestion...

**Review by fasterius — APPROVED** [review]
Looks good to me as well!

**Pull request merged**